docs/multicluster/setup-multi-primary.sh

```diff
@@ -110,6 +110,8 @@ spec:
         requests:
           cpu: 100m
           memory: 1024Mi
+      env:
+        ROOT_CA_DIR: /etc/cacerts
```

Comment:
Hmm. The default path for `ROOT_CA_DIR` is `./etc/cacerts` (defined [here](https://github.com/istio/istio/blob/492485ffbbaf21aebb468ba668aab340c13366eb/pilot/pkg/bootstrap/istio_ca.go#L81)), but we mount the `cacerts` secret into `/etc/cacerts` ([here](https://github.com/istio/istio/blob/master/manifests/charts/istio-control/istio-discovery/templates/deployment.yaml#L232)). 

I wonder why the default doesn't match. Maybe because the working dir used to be `/`, which made `./etc/cacerts == /etc/cacerts`, but that's no longer the case?